File: 2A_2/python/iloczyn_liczb.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
#
#  iloczyn_liczb.py
#  
#  Program liczy iloczyn 10 liczb pobranych od użytkownia.
#
#  Przykład algorytmu iteracyjnego

def main(args):
    iloczyn = 1
    for i in range(10):
        a = int(input("Podaj liczbę: "))
        iloczyn = iloczyn * a
        
    print(iloczyn)

    return 0

if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))

File: 2A_2/cpp/iloczyn_liczb.cpp
/*
 * iloczyn_liczb.cpp
 */


#include <iostream>

using namespace std;

int main(int argc, char **argv)
{
	int iloczyn, i, a;
	iloczyn = 1;
	
	for (i = 0; i < 10; i++) {
		cout << "Podaj liczbę: ";
		cin >> a;
		iloczyn = iloczyn * a;
	}
	
	cout << iloczyn;
	
	return 0;
}

